lib/api/sfApi.js

```diff
@@ -214,34 +78,56 @@ async function createReconciliationText(
   }
 }
 
-async function fetchReconciliationTexts(firmId, page = 1, refreshToken = true) {
-  setAxiosDefaults(firmId);
+async function readReconciliationTexts(firmId, page = 1, refreshToken = true) {
+  apiUtils.setAxiosDefaults(firmId);
```

Comment:
I think we can actually remove the need to set the default settings each time with a function and create an axios instance in the utils that already has the default settings and send around that instance instead. 

The setAxiosDefaults function is pretty ingrained already though in our code so would be need to test everything if we change it, but I think using an instance would be a perfect use-case for what we're doing here.

Reply:
I agree, but the need to use the `firmId` which can be provided via the CLI is the complex thing here. I guess we could modify it to set the firm id and do the check of the missing credentials somewhere else (directly by the cli as a first step).
But it sill be a big change, maybe we should tackle in a separate PR after.